CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # (Unreleased; add upcoming change notes here)
 
+# unreleased
+
```

Comment:
There is an unreleased header just above, no need to add this :)

Reply:
D'oh!